src/LinqKit.Core/ExpressionStarter.cs

```diff
@@ -67,6 +67,16 @@ internal ExpressionStarter(Expression<Func<T, bool>> exp) : this(false)
             return (IsStarted) ? _predicate = Predicate.And(expr2) : Start(expr2);
         }
 
+        /// <summary>Not</summary>
+        public Expression<Func<T, bool>> Not()
+        {
+            if (IsStarted)
```

Comment:
Can you please use `{` and `}` for if-statements?

Reply:
Sure.